apps/functions-subscription/src/use-cases/process-activation-job.ts

```diff
@@ -0,0 +1,40 @@
+import * as RTE from 'fp-ts/lib/ReaderTaskEither';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as RA from 'fp-ts/lib/ReadonlyArray';
+import { pipe } from 'fp-ts/lib/function';
+import { Capabilities } from '../domain/capabilities';
+import { ActivationJobItem } from '../domain/activation';
+
+type Env = Pick<Capabilities, 'activationConsumer'>;
+
+export const processActivationJob = (
+  job: ActivationJobItem,
+  maxChunkSize: number,
+) =>
+  pipe(
+    RTE.ask<Env>(),
+    RTE.flatMapTaskEither(({ activationConsumer }) =>
+      pipe(
+        activationConsumer.fetchActivationRequestItemsToActivate(job),
```

Comment:
**thought**: With this function signature, handling a large number of users to enable (e.g., 10K) all at once might pose problems. How about setting a limit on the maximum number of items to process instead? This way, we can focus on determining the number of items to handle per execution, without worrying about the chunk size. What do you think?